[2025-03-12T15:05:37.750+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-12T15:05:37.818+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: validate_all_tables.staffs scheduled__2025-03-05T13:00:00+00:00 [queued]>
[2025-03-12T15:05:37.854+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: validate_all_tables.staffs scheduled__2025-03-05T13:00:00+00:00 [queued]>
[2025-03-12T15:05:37.855+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-12T15:05:37.913+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): staffs> on 2025-03-05 13:00:00+00:00
[2025-03-12T15:05:37.933+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1312) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-12T15:05:37.939+0000] {standard_task_runner.py:72} INFO - Started process 1324 to run task
[2025-03-12T15:05:37.941+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'validate_all_tables', 'staffs', 'scheduled__2025-03-05T13:00:00+00:00', '--job-id', '487', '--raw', '--subdir', 'DAGS_FOLDER/s3_data.py', '--cfg-path', '/tmp/tmpgetdcem4']
[2025-03-12T15:05:37.944+0000] {standard_task_runner.py:105} INFO - Job 487: Subtask staffs
[2025-03-12T15:05:38.071+0000] {task_command.py:467} INFO - Running <TaskInstance: validate_all_tables.staffs scheduled__2025-03-05T13:00:00+00:00 [running]> on host b404e9833d11
[2025-03-12T15:05:38.315+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='validate_all_tables' AIRFLOW_CTX_TASK_ID='staffs' AIRFLOW_CTX_EXECUTION_DATE='2025-03-05T13:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-05T13:00:00+00:00'
[2025-03-12T15:05:38.319+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-03-12T15:05:38.320+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-03-12T15:05:38.321+0000] {logging_mixin.py:190} INFO - Current task name:staffs state:running start_date:2025-03-12 15:05:37.820054+00:00
[2025-03-12T15:05:38.322+0000] {logging_mixin.py:190} INFO - Dag name:validate_all_tables and current dag run status:running
[2025-03-12T15:05:38.323+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-12T15:05:39.293+0000] {httpchecksum.py:481} INFO - Skipping checksum validation. Response did not contain one of the following algorithms: ['crc32', 'sha1', 'sha256'].
[2025-03-12T15:05:39.340+0000] {logging_mixin.py:190} INFO - Validation successful. Found 10 staff records.
[2025-03-12T15:05:39.344+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-12T15:05:39.388+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-12T15:05:39.391+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=validate_all_tables, task_id=staffs, run_id=scheduled__2025-03-05T13:00:00+00:00, execution_date=20250305T130000, start_date=20250312T150537, end_date=20250312T150539
[2025-03-12T15:05:39.539+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-03-12T15:05:39.549+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-03-12T15:05:39.551+0000] {logging_mixin.py:190} INFO - Dag name:validate_all_tables queued_at:2025-03-12 15:04:59.240294+00:00
[2025-03-12T15:05:39.552+0000] {logging_mixin.py:190} INFO - Task hostname:b404e9833d11 operator:PythonOperator
[2025-03-12T15:05:39.713+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-12T15:05:39.773+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
